src/reference/config/bindings/.partials/options-kafka-topics.md

```diff
@@ -0,0 +1,29 @@
+#### topics[].name\*
+
+> `string`
+
+Topic name.
+
+#### topics[].key
+
+> `object`
+
+Enforce validation for key
+
+#### key.model
```

Comment:
Is this `key.model` expected to be at the same indentation / header level as its parent `topics[].key`?

Reply:
yes indention levels needed to be normalized at 3 and 4 for imports to be used effectively. so root properties will all have lvl 3 headers and everything else will have 4.